File: myBlogSystem/hero-parent/hero-web/src/main/webapp/js/service/blogService.js
app.service('blogService',function($http){
	//根据分类ID查询广告列表
	this.findAll=function(){
	    return $http.get('../blog/findAll.do');
	}	
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../blog/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../blog/findOne.do?id='+id);
	}
	//查询博客实体
	this.findBlog=function(id){
		return $http.get('../blog/findBlog.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../blog/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../blog/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../blog/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../blog/search.do?page='+page+"&rows="+rows, searchEntity);
	}   
});

File: myBlogSystem/hero-parent/hero-interface/src/main/java/club/hero/service/BlogService.java
package club.hero.service;

import java.util.List;

import club.hero.pojo.Blog;
import entity.PageResult;



	public interface BlogService {
	
	/**
	 * 查询所有博客文章
	 * @return
	 */
	public List<Blog> findAll();


	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(Blog blog);
	
	
	/**
	 * 修改
	 */
	public void update(Blog blog);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public Blog findOne(Integer id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Integer [] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(Blog blog, int pageNum,int pageSize);
	
	

	
	
	


}